sc-application/src/main/java/pl/agh/shopping/card/application/service/ShoppingCardItemService.java

```diff
@@ -37,6 +37,7 @@ public ShoppingCardItemResponseDTO add(Long shoppingCardId, ShoppingCardItemRequ
         shoppingCardItem.setShoppingCard(shoppingCart.get());
 
         var bookInfo = getBookInfo(shoppingCardItem);
+        shoppingCardItem.setActual_price((float) bookInfo.get("price"));
```

Comment:
Z jakiegoś powodu ten get zwraca double a nie floata. Zrób coś takiego:
```suggestion
               Double price = (Double) bookInfo.get("price");
        shoppingCardItem.setActual_price(price.floatValue());
```